src/components/appMenuSections/dashboard/Dashboard.tsx

```diff
@@ -21,8 +20,8 @@ import { Box } from '@mui/material';
 const Dashboard = () => {
   const [authenticatedSuccessfully, setAuthenticatedSuccessfully] = useState<boolean>(false);
   const [clickedJoinPoolButtonByViewer, setClickedJoinPoolButtonByViewer] = useState<boolean>(false);
-  const [currentNotifier, setCurrentNotifier] = useState<string>('');
-  const [minersList, setMinersList] = useState<any>([]);
+  const [currentNotifier, setCurrentNotifier] = useState<string | null>(null);
```

Comment:
is it the best way to initialize it? (before taking the value from the smart contract request)

Reply:
We'd need to have a way of doing an async loading for some of these. https://github.com/stacks-degens/starters-front-end/issues/55